STYLE_GUIDE.md

```diff
@@ -15,7 +15,7 @@ void formatted_code_again;
 ```
 
 #### Python
-Python code should conform to [PEP8](https://www.python.org/dev/peps/pep-0008/).
+Python code should conform to [PEP8](https://www.python.org/dev/peps/pep-0008/) and [TensorFlow Community Python Style Guide] (https://www.tensorflow.org/community/contribute/code_style).
```

Comment:
Hi, Armando. I'm afraid there is a conflict between the two files above. For python code style, we conform to PEP8, which might be slightly different with Google Python Style Guide.

Do you want to create a same name section which links to
https://www.tensorflow.org/community/contribute/code_style#tensorflow_conventions_and_special_uses ?

cc @seanpmorgan Sean, what do you think?

Reply:
Agree we can't link to the full style guide because we don't use it. The conventions and special uses can be linked in but I think the hyperlink should read `TensorFlow conventions` and not `Tensorflow style guide`

Reply:
I think I linked to TensorFlow Style Guide not Google Style guide but, TensorFlow Style Guide itself says this on the top of that guide : `Follow the PEP 8 Python style guide, except TensorFlow uses 2 spaces instead of 4. Please conform to the Google Python Style Guide`. So if PEP8 and Google and TensorFlow style guide, they all 3 conflict with each other, then do we need to fix that line in TensorFlow Style Guide as well ?

Reply:
We don't always conform those code styles of TensorFlow Style Guide, that's why we can't link to the full page. Take python as an example, for historical reasons, tensorflow doesn't conform to the new Google python code style, which might be compatible with PEP8. While for addons, we strictly stick to PEP8.

Sean's suggestion: `TensorFlow conventions` sounds better.

Reply:
Updated the doc to have a conventions sesction.